packages/loader/container-loader/src/containerStorageAdapter.ts

```diff
@@ -69,6 +84,7 @@ export class ContainerStorageAdapter implements IDocumentStorageService, IDispos
 		 * ArrayBufferLikes or utf8 encoded strings, containing blobs from a snapshot
 		 */
 		private readonly blobContents: { [id: string]: ArrayBufferLike | string } = {},
+		private pendingLocalGroupIdSnapshots: Record<string, ISnapshotInfo> | undefined,
```

Comment:
How about renaming it to: "pendingLocalGroupIdSnapshots" -> "loadingGroupIdSnapshotsFromLocalPendingState"? Just to make it more clear?

Reply:
Also add description in the comments above

Reply:
Sounds good